tests/test_splitter.py

```diff
@@ -8,8 +8,57 @@
 
 class TestSplitter(TestCase):
 
-    def test_Split(self):
-        sp.Splitter.Split("filepath")
+    def test_Split_one(self):
```

Comment:
might make sense to have it under one function. especially if you foresee having other delimiters showing up in the future.